sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css

```diff
@@ -39,6 +39,10 @@
   font-weight: bold;
 }
 
+#plan-viz-graph svg text {
```

Comment:
Previously, we handled all `.label`. Now, this PR is about `text`.
If we start to consider `text`, are you sure that this is the last `text`?

Reply:
I think we need to consider three part of DAG-viz. Stages, RDDs and query plans.

In the previous PR, I changed the color of `.label` for stages and RDDs. Both are covered by [here](https://github.com/apache/spark/pull/28352/files#diff-e5cdb3a11044297da1b3136680150173R22).
I also changed color of `.label` for query plans [here](https://github.com/apache/spark/pull/28352/files#diff-98200c641ee59851dddec016c9c67b7fR22).

About `text`, for stages and RDDs, it's already covered [here](https://github.com/apache/spark/blob/master/core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.css#L75).
So I think the last one is for query plans. It's covered by this PR.

Reply:
Thanks for the confirmation!